src/main/scala/nl.knaw.dans.easy.validatebag/rules/ProfileVersion0.scala

```diff
@@ -73,10 +73,9 @@ object ProfileVersion0 {
     NumberedRule("3.2.1", filesXmlConformsToSchemaIfFilesNamespaceDeclared(xmlValidators("files.xml")), dependsOn = Some("2.2(b)")),
     NumberedRule("3.2.2", filesXmlHasDocumentElementFiles, dependsOn = Some("2.2(b)")),
     NumberedRule("3.2.3", filesXmlHasOnlyFiles, dependsOn = Some("3.2.2")),
-
     NumberedRule("3.2.4", filesXmlFileElementsAllHaveFilepathAttribute, dependsOn = Some("3.2.3")),
     // Second part of 3.2.4 (directories not described) is implicitly checked by 3.2.5
-    NumberedRule("3.2.5", filesXmlAllFilesDescribedOnce, dependsOn = Some("3.2.4")),
+    NumberedRule("3.2.5", filesXmlAllFilesDescribedOnce, dependsOn = Some("1.1.1")),
```

Comment:
shouldn't it be also dependent on 3.2.4?

Reply:
That is correct, but you can give only one dependency.

Reply:
I was actually going to suggest that we would change the `dependsOn` parameter so that we can give a list of dependencies.
Even if we could solve this present problem in some other way, I can imagine that we will face other situations where it would we handy to be able to give more than one dependency.

Reply:
but even then, the dependency on 1.1.1 is too strong. 1.1.1 checks more than only the existence of the /data dir. 
if we make rule 3.2.5 depend on 1.1.1 it would follow that when a deposit does not comply with rule 1.1.1, for whatever reason, it won't be checked for this rule either. I don't like that. the only part of 1.1.1 we need here is the existence of the data dir.

Reply:
My original solution was to add a new rule (2.6), checking the existence of the data dir. And 3.2.5 was depending on this new rule. But @janvanmansum was of opinion that it was not necessary because of the existing rule 1.1.1

Reply:
I changed the type of `dependsOn` parameter from Option[RuleNumber] to List[RuleNumber].
I added (again:)) a new rule 2.6 to check the existence of the data directory.
For the rule 3.2.5 I set as dependencies rules 2.6 and 3.2.4

Reply:
if we call this new rule `1.1.1(b)` and the existing 1.1.1 gets the suffix (a), then we're set.
We don't have to introduce a new rule, that's already covered by an existing one. and we can make 2.3.5 depend to as little pre-conditions as possible.

Reply:
Changed the name of the new rule, checking existence of the datadir, to `1.1.1(datadir)`